autokeras/hypermodel/hyperblock.py

```diff
@@ -0,0 +1,94 @@
+from tensorflow.python.util import nest
+
+from autokeras.hypermodel import block
+from autokeras.hypermodel import node
+from autokeras.hypermodel import preprocessor
+
+
+class HyperBlock(block.Block):
```

Comment:
HyperBlock can contain Blocks inside, but Block cannot.
The purpose of separating HyperBlock from Block is to parse the graph for preprocessors.

After we have the preprocessing layers, we no longer need the HyperBlock.
Then any Block can contain Blocks inside since we do not need to parse the graph for preprocessing.

I will add the info in docstring.